performance_test/src/data_running/data_runner_factory.cpp

```diff
@@ -20,6 +20,7 @@
 #include <memory>
 
 #include "../communication_abstractions/ros2_callback_communicator.hpp"
+#include "../communication_abstractions/ros2_waitset_communicator.hpp"
```

Comment:
do you have to use relative paths instead of just #include "ros2_waitset_communicator.hpp"?

Reply:
Okay I was just being consistent with the already imported `ros2_callback_communicator.hpp`. Should I make this change?

Reply:
I think it's not in the scope of this PR, I see it's in all the files done like this, so maybe we stay consistent.